### ISSUE TITLE
Caption mismatch on bug report template

### ISSUE THREAD

**Issue by dstein64** [vim/vim#9329]
### Steps to reproduce

When filing Issue #9328, I noticed that the caption under "Version of Vim" mentions "development environment" with an example of "GNOME Terminal".

![version_of_vim](https://user-images.githubusercontent.com/541289/145688220-ffd29b89-3a3d-48f6-a993-705d9c726c39.png)

### Expected behaviour

Not applicable

### Operating system

Not applicable

### Version of Vim

Not applicable

**Comment by brammool** [comment]
So, what should it be?  Wikipedia has GNOME in capitals: https://en.wikipedia.org/wiki/GNOME_Terminal

**Comment by dstein64** [comment]
The title for the field is "Version of Vim" but the caption corresponds to a development environment.

I think the caption should be something like:
"The version of Vim you're using. [e.g.,  8.2 w/ patches ... from Ubuntu repository, or 8.2.3784 compiled from source]"

**Comment by dstein64** [comment]
I now see my misunderstanding. I thought "version" in the caption corresponds to the terminal version, not the Vim version.

Adding "Vim" prior to version might help (e.g., "The name of your development environment, Vim version..."), but if no one else misinterpreted then I guess it's not an issue.

**Comment by lacygoill** [comment]
I noticed that too.  The description does not seem to match the caption.

Besides, asking for the Vim version is not enough.  When asked for a Vim version, most of the time, people answer something like 8.2.  But that's often not enough, because it does not include the patch level.
We could ask for `v:versionlong` (available since [8.1.1526](https://github.com/vim/vim/releases/tag/v8.1.1526)), but that might be misleading because there might be gaps in the list of included patches.  `$ vim --version | grep patches` seems more reliable.

Beyond that, we could also ask for the name of the terminal emulator and for `$TERM`.  For some issues, this kind of information is missing, but needed to find a solution.
```diff
diff --git a/.github/ISSUE_TEMPLATE/bug_report.yml b/.github/ISSUE_TEMPLATE/bug_report.yml
index b4e3128f3..c0aa08a20 100644
--- a/.github/ISSUE_TEMPLATE/bug_report.yml
+++ b/.github/ISSUE_TEMPLATE/bug_report.yml
@@ -37,7 +37,24 @@ body:
     attributes:
       label: Version of Vim
       description: >
-        The name of your development environment, version. [e.g. GNOME Terminal (v0.0.0000)]
+        The version of your Vim binary including all of its patches (as given by `$ vim --version | grep patches`).
+        For example, 8.2.1234
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Terminal name and version
+      description: >
+        The name and version of the terminal emulator in which you run Vim.
+        For example, GNOME Terminal 3.36.2
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: TERM environment variable
+      description: >
+        The value of the TERM environment variable (as given by `$ echo $TERM`).
+        For example, xterm-256color
     validations:
       required: true
   - type: textarea
```

**Comment by brammool** [comment]
Ah, the example does not match the header.
I like to keep this mostly free-form, but we can  mention a few things.
